components/dashboard/src/workspaces/BrowserExtensionBanner.tsx

```diff
@@ -101,7 +114,8 @@ export function BrowserExtensionBanner() {
 
     useEffect(() => {
         const installedOrDismissed =
-            sessionStorage.getItem("browser-extension-installed") ||
+            sessionStorage.getItem("browser-extension-installed") || // todo(ft): delete after migration is complete
```

Comment:
While this will fix issue looking forward, we could introduce an additional safe-guard: Delay the execution of this useEffect why some time (2s?) to make it less likely we are loosing this race. WDYT? :thinking:

Reply:
I have not done this primarily for two reasons:
1. The rendering of the nudge would be delayed, most often more than it has to
2. It makes an assumption about the client's networking setup and the arbitrary two seconds could still not be enough on some poor cellular connections

I am playing around with the idea of using a `MutationObserver` instead to be able to listen to changes WRT the extension being active. It could work if we have a special element indicating this state and listen for changes to it from the Dashboard, mutating it from the extension.